scripts/python/subset_material_icons.py

```diff
@@ -0,0 +1,95 @@
+import os
+import re
+import requests
+from bs4 import BeautifulSoup
+
+from lib import subset_font
+
+def parse_html_file(file_path):
+  with open(file_path, 'r') as file:
+    soup = BeautifulSoup(file.read(), 'html.parser')
+    icons = soup.find_all(class_='material-icons')
```

Comment:
material-icons is a tailwind utility, which can be used with `@apply`. So this doesn't work. Maybe consider one of the following?
1. Maybe we can simply search for all existence of defined icon names in the files (html, css, js, json), not limited to class names. This could cause false matches (For example, "phone" is too common that it always appear, which may not be used as icon). But I believe even with the false matches, the subset is small enough.
2. We can simply define a list of icon name. And we manually add icons there.

Reply:
Yes that's why I also parse_css_file as well to cover this case. 

Nevertheless, maybe I added a custom list if icon names that developers can add if they found auto-parse failed to include the glyph.

Reply:
Added in 41a8e5d

Reply:
Since we are using ligature to apply the icon, if we can get the full list of ligature keywords. Then we only need to scan resources/**/*.{html,css,ts} for those keywords. This algorithm is much simpler. (I remember tailwind is doing something similar, instead of really parsing the HTML, CSS, JS files)

Reply:
I guess this is the list? https://github.com/google/material-design-icons/blob/master/variablefont/MaterialSymbolsOutlined%5BFILL%2CGRAD%2Copsz%2Cwght%5D.codepoints

Reply:
Updated in 2d5f8a1